v1.1-beta/matrixx-digital-commerce/matrixx-tmf-party-management/cnf-testsuite.yml

```diff
@@ -2,7 +2,7 @@
 helm_directory: "tmf-service-activation-management"
 release_name: "cnf"
 helm_install_namespace: "matrixx"
-helm_values: "--values /home/dwilmes/.mtx/konstruxx/working/values-232795837.yaml\
+helm_values: "--values ~/.mtx/konstruxx/working/values-232795837.yaml\
   \ --devel"
```

Comment:
@daniel-wilmes @deerowbear Are end-users expected to use the --devel flag?

Reply:
@taylor removed -devel